app/src/main/java/net/squanchy/typeface/TypefaceController.kt

```diff
@@ -0,0 +1,26 @@
+package net.squanchy.typeface
+
+import android.graphics.Typeface
+import android.text.Spannable
+import android.text.SpannableString
+import android.text.Spanned
+
+class TypefaceController {
+
+    fun applyTypeface(text: CharSequence, typeface: Typeface) =
```

Comment:
This could be an extension function of `CharSequence`, something like `fun CharSequence.applyTypeface(typeface: Typeface)`